docs/source/getting-started/content/proc_accessing-the-openshift-cluster.adoc

```diff
@@ -32,6 +32,14 @@ $ crc oc-env
 
   . Run the printed command.
 
+  . If you haven't accessed the cluster using the [command]`oc` yet, log in by running the [command]`oc login -u kubeadmin https://api.crc.testing:6443` command.
```

Comment:
We should avoid using "if" whenever possible. We can simply instruct the user to log in to the cluster via `oc`.

Reply:
If I remove this, could people end up running "oc login" all the time? I guess not?

Reply:
Personally, I doubt it. :)